src/test/java/keycontacts/testutil/FindStudentDescriptorBuilder.java

```diff
@@ -0,0 +1,71 @@
+package keycontacts.testutil;
+
+import keycontacts.logic.commands.FindCommand.FindStudentDescriptor;
+import keycontacts.model.student.Student;
+
+/**
+ * A utility class to help with building FindStudentDescriptor objects.
+ */
+public class FindStudentDescriptorBuilder {
+    private FindStudentDescriptor descriptor;
+
+    public FindStudentDescriptorBuilder() {
+        descriptor = new FindStudentDescriptor();
+    }
+
+    public FindStudentDescriptorBuilder(FindStudentDescriptor descriptor) {
+        this.descriptor = new FindStudentDescriptor(descriptor);
+    }
+
+    /**
+     * Returns a {@code EditStudentDescriptor} with fields containing
```

Comment:
All javadocs in this file need to be fixed, replace `EditStudentDescriptor` with `FindStudentDescriptor`